### PR TITLE
Scrum 144 - PATCH endpoint to modify the lostbObject Status

### PR DESCRIPTION
**Pull request by JOSEEAND**
[evidences.pdf](https://github.com/user-attachments/files/16565174/evidences.pdf)

### CODE REVIEW

**Review by LilMat69 — CHANGES_REQUESTED** [review]
Please add this endpoint to the swagger file, remember the authorization header in the swagger as well for the permissions

**Comment by JOSEEAND** [comment]
image

File: swagger.yml
Comment (LilMat69) [inline swagger.yml:1]: Add the following messages into swagger: 

image

image

**Review by LilMat69 — CHANGES_REQUESTED** [review]

File: swagger.yml
Comment (JOSEEAND) [reply swagger.yml:1]: image

**Review by LilMat69 — APPROVED** [review]

**Pull request merged**